src/tests/electron/tests/first-time-dialog.test.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { createApplication } from 'tests/electron/common/create-application';
+import { scanForAccessibilityIssues } from 'tests/electron/common/scan-for-accessibility-issues';
+import { AppController } from 'tests/electron/common/view-controllers/app-controller';
+import { DeviceConnectionDialogController } from 'tests/electron/common/view-controllers/device-connection-dialog-controller';
+
+describe('first time dialog', () => {
+    let app: AppController;
+    let dialog: DeviceConnectionDialogController;
+
+    beforeEach(async () => {
+        app = await createApplication({ suppressFirstTimeDialog: false });
+        dialog = await app.openDeviceConnectionDialog();
+    });
+
+    afterEach(async () => {
+        dialog = null;
```

Comment:
This line doesn't seem necessary with dialog being set every time in beforeEach but I might be missing something here ...

Reply:
I saw this in the other e2e test setups and assumed it helped cleanly end the webdriver process, but I can't find a reference now. I'll merge for now & check with the team